### PR TITLE
Added a features section to `understand`

### PR DESCRIPTION
**Pull request by bendecoste**
This is some quick (and basic) documentation for
- auditing
- policy
- projects
- roles
- users

image

### CODE REVIEW

**Comment by bendecoste** [comment]
@kjam Would you be able to run through https://deploy-preview-50--cp-docs-demo.netlify.app/understand/features/ and provide some feedback? Things we would like to add etc?

File: docs/understand/features/users.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Users
+
+Users in cape exists much like users in other applications. All actions performed
```
Comment (justin1121) [inline docs/understand/features/users.md:3]:
```suggestion
Users in Cape are much like users in other applications. All actions performed
```

File: docs/understand/features/projects.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Projects
+
+Projects are the building block for Policy within Cape. In the real world, most
+data access is granted depending on the use case and what the user is trying
+to achieve. Cape groups all of those different use cases into projects.
+
+For example, if you are a data scientist working for a bank in department A who is
```
Comment (justin1121) [inline docs/understand/features/projects.md:7]:
```suggestion
For example, if you are a data scientist working for a bank in the fraud department who is
```

File: docs/understand/features/projects.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+tasked with creating a model to detect fraud. You may be able to train a better
+model if you can use other customer data that belongs to department B. In this context,
```
Comment (justin1121) [inline docs/understand/features/projects.md:9]:
```suggestion
model if you can use other customer data that belongs to the sales department. In this context,
```

File: docs/understand/features/auditing.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Audit Log
+
+All user actions in Cape are captured and viewable in the audit log. Examples
+of a tracked action include getting a [policy](./policy.md), logging in,
+creating a new [project](./project.md), etc.
```
Comment (justin1121) [inline docs/understand/features/auditing.md:5]:
```suggestion
and creating a new [project](./project.md).
```

**Comment by kjam** [comment]
Looks great! A few notes (not holding us up but might want to circle back on later):

- We should likely eventually expand the policy description a bit since I think there's likely a bit more of a story to tell there!
- Can we add some details on states of projects to help guide people on the project lifecycle?
- Cape is not consistently capitalized, which makes me go crazy :sob: 
- The roles are a bit confusing right now upon first glance, I'll focus some User Research on them but we might want to open a separate issue.
- Do we have more documentation on how audit logs work? Where are they stored? Can people set deletion or rotation? Might be useful to add a bit more details on how it works.

**Pull request merged**